src/main/webapp/index-script.js

```diff
@@ -314,3 +314,66 @@ function generateResult(place) {
   resultEntry.appendChild(resultGrid);
   return resultEntry;
 }
+
+/** Fetch Reviews
+ * Queries ReviewServlet with elementID to find internal datastore
+ */
+function fetchReviews(elementID) {
+  console.log("Fetching reviews for ID: #" + elementID);
+  fetch('/review').then(response => response.json()).then((reviewsArr) => {
+    populateReviews(reviewsArr);
+  });
+}
+
+function activateReviewModal() {
+  // TODO: handoff between modals
+  // Activate the review modal
+}
+
+/** Creates a structure to put reviews in modal
+ * Takes in array of JS reviews
+ */
+function populateReviews(reviewList) {
+  Console.log('Populating reviews in modal...');
+  const listContainer = document.getElementById('reviews-list-container');
+  const entireList = document.createElement('ul');
+  entireList.id += 'reviews-list';
+
+  if (reviewList.asSingleElement() == null) {
+    entireList.appendChild(noReviews());
+  } else { 
+    reviewList.forEach(review => {
+      entireList.appendChild(generateReview(review));
+    });
+  }
+
+  listContainer.appendChild(entireList);
+}
+
+/** Creates a review element using grid styling
+ * Puts the review text in a <p> element
+ */
+function generateReview(review) {
+  const reviewEntry = document.createElement('li');
+
+  const reviewGrid = document.createElement('div');
+  reviewGrid.className += review-grid;
+
+  const reviewText = document.createElement('p');
+  reviewText.innerHTML += "review.text placeholder = Great food and even better service.  Beer pizza Italian Beef. Who could ask for more.";
```

Comment:
Yuck! Beer pizza italian beef does not sound appetizing!

Reply:
The power of commas...

Reply:
Boilerplate review from Aurelio's Pizza in Mokena. 
_American Heartland Cuisine_
**That is vestigial testing code.**

Reply:
### Cool ### 
**COOL**
_COOL_